client/components/badge/index.jsx

```diff
@@ -1,4 +1,4 @@
-import classNames from 'classnames';
+import { Badge as BadgeWrapper } from '@automattic/components';
```

Comment:
Can we make this an even thinner wrapper using an approach something like the following?
```suggestion
import { Badge } from '@automattic/components';

export default Badge;
```

And then delete everything else in `client/components/badge/` except for `index.jsx` (which we could also rename to `index.tsx` if we want to).